src/movies/movies/src/app/movies/movies.component.ts

```diff
@@ -1,30 +1,24 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
-import { MovieType } from '.././app.component';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Movie } from '../models';
+
+interface ChangedMovie {
+  movie: Movie;
+  index: number;
+}
 
 @Component({
   selector: 'app-movies',
   templateUrl: './movies.component.html',
   styleUrls: ['./movies.component.css']
 })
-export class MoviesComponent implements OnInit {
-  @Output() arrayChanged = new EventEmitter<Array<MovieType>>();
-  constructor() { }
-
-  @Input() movies!: Array<MovieType>;
-
-  ngOnInit(): void {
-  }
-  changedMovie!: ChangedMovieType
-  
+export class MoviesComponent {
+  @Input() movies!: Array<Movie>;
```

Comment:
In TS it's common practice to indicate array types like `Movie[]` not `Array<Movie>`. Please change here and in other cases